lib/SIL/Utils/SILBridging.cpp

```diff
@@ -110,8 +110,8 @@ void registerBridgedClass(BridgedStringRef bridgedClassName, SwiftMetatype metat
   SILNodeKind kind = iter->second;
   SwiftMetatype existingTy = nodeMetatypes[(unsigned)kind];
   if (existingTy) {
-    llvm::errs() << "Double registration of class " << className << '\n';
-    abort();
+    // We have already registered this class
```

Comment:
@egorzhdan @eeckstein 
What's the possible harm in doing this? 
We have a library (`libSwiftScan`) that gets loaded from a long-running process which needs to occasionally initialize a new instance of the tool the library provides (`swiftscan_scanner_create` in this same PR diff), which needs the ability to `initializeSwiftModules`. The same place currently calls `INITIALIZE_LLVM` and that seems to support doing nothing on repeated initialization calls.

Reply:
I added this check to catch possible inconsistencies when registering classes. Your change is fine, because currently all the registration can happen multiple times. Though, it would be somehow nicer to keep this check and guard the initialization call with
```
if (!swiftModulesInitialized())
  initializeSwiftModules();
```